docs/emr/configuration-and-execution.rst

```diff
@@ -0,0 +1,16 @@
+===========================
+Configuration and Execution
+===========================
+
+- Configuring Presto Admin on EMR cluster should proceed in the same manner as on any other cluster (see :ref:`presto-admin-configuration-label`) except for the following caveat:
+
+	- We recommend designating the EMR master node as the Presto coordinator during Presto Admin configuration.
+
+- Running Presto Admin on EMR cluster should proceed in the same manner as on any other cluster (see sections starting from :ref:`presto-server-installation-label`  onwards in :ref:`quick-start-guide-label`) except for the following caveats:
+
+	- The default version of Java installed on EMR cluster (up to EMR 4.4.0) is 1.7, whereas Presto requires Java 1.8. Install Java 1.8 on EMR cluster by following the instructions in Java installation (see :ref:`java-installation-label`).
```

Comment:
I am going to add this in troubleshooting section with the error message. But we should add it here too.